contracts/modules/RelayerManager.sol

```diff
@@ -387,25 +370,24 @@ abstract contract RelayerManager is BaseModule {
         assembly { chainId := chainid() }
     }
 
-    function getRequiredSignaturesToStartSession(address _wallet) internal view returns (uint256, OwnerSignature) {
-        // owner  + [n/2] guardians
-        uint numberOfGuardians = Utils.ceil(guardianStorage.guardianCount(_wallet), 2);
-        require(numberOfGuardians > 0, "RM: no guardians set on wallet");
-        uint numberOfSignatures = 1 + numberOfGuardians;
-        return (numberOfSignatures, OwnerSignature.Required);
+    function startSession(address _wallet, address _sessionUser, uint64 _duration) external onlyWallet(_wallet) returns (bool)
+    {
+        require(_sessionUser != address(0), "RM: Invalid session user");
+        require(_duration > 0, "RM: Invalid session duration");
+
+        uint64 expiry = Utils.safe64(block.timestamp + _duration);
+        sessions[_wallet] = Session(_sessionUser, expiry);
+        emit SessionCreated(_wallet, _sessionUser, expiry);
     }
 
-    function startSession(address _wallet, bytes calldata _data) internal returns (bool) {
-        require(_data.length >= 68, "RM: Invalid session parameters");
-        (address key, uint64 expires) = abi.decode(_data[36:], (address, uint64));
-        if (key != address(0)) {
-            if (expires > 0) {
-                sessions[_wallet] = Session(key, expires);
-                emit SessionCreated(_wallet, key, expires);
-            }
-            return true;
-        }
-        return false;
+    /**
+    * @notice Clears the active session of a wallet if any.
+    * @param _wallet The target wallet.
+    */
+    function clearSession(address _wallet) external onlySelf()
+    {
+        delete sessions[_wallet];
+        emit SessionCleared(_wallet, sessions[_wallet].key);
```

Comment:
The second argument logged will always be `address(0)` since the session has been deleted.